### ISSUE TITLE
Feature request: Exhibit Builder 3.0 layout for maps. 

### ISSUE THREAD

**Issue by sheepeeh** [performant-software/Neatline#293]
Like the one available for Timelines.

**Comment by waynegraham** [comment]
This is great @sheepeeh...just adding a few notes below...

ExhibitBuilder 3 allows plugins to define custom layouts (see http://omeka.readthedocs.org/en/eb3/Tutorials/extendingExhibitBuilder.html). John had made a pull request for NeatlineTime to do this (https://github.com/scholarslab/NeatlineTime/commit/8ee3755cee9e02510fde7e9d250626b8b4594579), but I suspect this may be a bit more complicated because of how the views are constructed in Neatline.

At a bear minimum, it could probably use the `fullscreen` view of exhibit embedded with an `iframe`. I think it quickly gets a bit more complicated from there, but we may need to take some time to look at what this feature would entail.

**Comment by sheepeeh** [comment]
That's actually how I do it now--manually adding an iframe with no border within a text block. Just being able to select the map from a list to do this rather than finding the appropriate link would be helpful.

**Comment by davidmcclure** [comment]
@erochest, was this resolved by the work you pushed up a couple days ago?

**Comment by erochest** [comment]
Yeah, github shows that it will close once 6e105de34ea73fc9572767278380b8c9cf40c603 is merged into master.

**Comment by davidmcclure** [comment]
Oh cool, sorry, didn't see that.
